tracks.yaml

```
@@ -0,0 +1,26 @@
+tracks:
+  dashing:
+    actions:
+    - bloom-export-upstream :{vcs_local_uri} :{vcs_type} --tag :{release_tag} --display-uri
+      :{vcs_uri} --name :{name} --output-dir :{archive_dir_path}
+    - git-bloom-import-upstream :{archive_path} :{patches} --release-version 0.1.0+osrf202@gb421e10
+      --replace
+    - git-bloom-generate -y rosrelease :{ros_distro} --source upstream -i :{release_inc}
+    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+      -i :{release_inc} --os-name ubuntu
+    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+      -i :{release_inc} --os-name debian --os-not-required
+    - git-bloom-generate -y rosrpm --prefix release/:{ros_distro} :{ros_distro} -i
+      :{release_inc}
+    devel_branch: master
+    last_release: b421e1039ffb08eada3a9b75f3c6a581a41962c4
+    last_version: 0.1.0
+    name: fastrtps
```

```suggestion
    name: cyclonedds
```

This name is essentially "internal" to bloom but this would be v confusing.

---

Wow, I can't believe I missed that. Good catch.

---

Updated.